client/boot/index.js

```diff
@@ -52,6 +53,9 @@ function init() {
 
 	debug( 'Starting Calypso. Let\'s do this.' );
 
+	// prune sync-handler records
+	syncHandler.pruneRecordsFrom();
```

Comment:
> The function name really reads like it should be passed an argument.

That's 100% right. Now I wonder how to avoid define `2 days`:

```es6
const DAY_IN_HOURS = 24;
const HOUR_IN_MS = 3600000;
export const LIFETIME = 2 * DAY_IN_HOURS * HOUR_IN_MS;
```

Maybe it's time to define a module with a set of constants.